doc/source/data/dataset.rst

```diff
@@ -239,7 +244,7 @@ By default, transformations are executed using Ray tasks. For transformations th
     class BatchInferModel:
         def __init__(self):
             self.model = ImageNetModel()
```

Comment:
I'm assuming that actually defining `ImageNetModel` would be overkill, and wouldn't be a net benefit for the example.

If we wanted to define it, we could do something like:
```python
import torch

class ImageNetModel:
    def __init__(self):
        self.model = resnet50(pretrained=True).eval().half().cuda()

    def __call__(self, input_tensor_np):
        input_tensor = torch.from_numpy(input_tensor_np).half().cuda()
        with torch.no_grad():
            output_tensor = self.model(input_tensor)
            result = torch.argmax(output_tensor, dim=1).cpu()
        return result.numpy()
```

Reply:
We could definitely use such an example. How about after we add the tensor types support? Then it could be end to end (data prep -> loading -> model) with a tensor value type dataset.